manual_cases.md

```
@@ -0,0 +1,533 @@
+| Field | Value |
+|-------|-------|
+| **ID** | OCP-73854 |
+| **Version** | 4.16, 4.17, 4.18, 4.19 |
+| **Assignee** | Nidan Gavali (ngavali) |
+| **Author** | Ying Zhou (yinzhou) |
+| **Importance** | Low |
+| **Test Type** | Functional |
+
+#### Setup
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)